### ISSUE TITLE
 /v2/ledgers/hist endpoint returns only last two years of ledger entries

### ISSUE THREAD

**Issue by MaxPovver** [bitfinexcom/bitfinex-api-node#398]
We use /v2/ledgers/hist to get ledger data for our clients.
However whatever parameters we pass there, there is no way to get ledger history older then august 2016. however user has trades in 2014, so ledger entries for 2014 must also be there
What I call 'ledgers/hist' with parameters 'limit' => 5000
What expect to see:
all 5000 ledger entries
In fact I receive only last 3700 entries, first in august 2016
setting start/end parameters didn't check anything, parameter skip also didn't help.
Api key with issue G6muxQLbRiyU94....
I use my own lib

**Comment by MaxPovver** [comment]
UPD:
got response from bitfinex support after almost 2 months waiting, they say it is expected because of servers migration and will last up to 2 months. Also they said it will be limited to 3 months for now:

> Apologies for the delayed response. We are currently in the process of migrating servers. As of right now, you will only be able to retrieve ledger entry history as far back as the past 3 months. Once our server migration is complete you should be able to retrieve the data you specified earlier. We hope to fully complete this within the next two months. And apologies for the inconvenience.

**Comment by f3rno** [comment]
@MaxPovver any difference now?

**Comment by MaxPovver** [comment]
@f3rno yes, now it finally returns all data, thanks! Although I am 99% it is still missing some initial deposits. Because for one of our users, his USD ledger history starts from [....,"USD",null,1449133933000,null,-3.6233,18620.66915659,null,"Trading fees for 10.0 BTC @ 362.33 on BFX (0.1%) on wallet exchange"]]
And his BTC ledger history starts from:
[....,"BTC",null,1449133934000,null,10,10,null,"Exchange 10.0 BTC for USD @ 362.33 on wallet exchange"]]
And so it looks like he didn't get his BTC using deposit, getting them out of thin air instead.

**Comment by MaxPovver** [comment]
Thanks for your help anyway!
